daemon/info_unix.go

```diff
@@ -56,6 +51,21 @@ func (daemon *Daemon) FillPlatformInfo(v *types.Info, sysInfo *sysinfo.SysInfo)
 		v.RuncCommit.ID = "N/A"
 	}
 
+	v.ContainerdCommit.Expected = dockerversion.ContainerdCommitID
+	if rv, err := exec.Command("docker-containerd", "--version").Output(); err == nil {
```

Comment:
Can we let `docker-containerd --version` print JSON?

Reply:
That's a request for https://github.com/containerd/containerd ;-)

Reply:
opened https://github.com/containerd/containerd/issues/1561